src/dnx/dnx.cpp

```diff
@@ -281,6 +281,7 @@ int CallApplicationProcessMain(int argc, dnx::char_t* argv[], dnx::trace_writer&
         _X("dnx.coreclr.so");
 #else
         _X("dnx.clr.dll");
+        SetEnvironmentVariable(_X("DNX_RUNTIME_OS"), _X("1"));
```

Comment:
Since the flag indicates that the OS is Windows, perhaps a better name would be `DNX_IS_WINDOWS` or something? I spent a few minutes trying to figure out how `1` could be the name of an OS :).